web/app/js/components/Navigation.jsx

```diff
@@ -114,6 +112,7 @@ const styles = theme => {
       width: `${navLogoWidth}px`,
     },
     namespaceChangeButton: {
+      backgroundColor: grey[400],
```

Comment:
I think we can style a little bit the button to look similar to previous one: 
```
    namespaceChangeButtonInputRoot: {
      backgroundColor: grey[300],
      boxShadow: "rgba(0, 0, 0, 0.2) 0px 3px 1px -2px, rgba(0, 0, 0, 0.14) 0px 2px 2px 0px, rgba(0, 0, 0, 0.12) 0px 1px 5px 0px",
      borderRadius: "5px",
      padding: "4px 12px !important",
      border: 0,
      "&:hover": {
        borderColor: "transparent",
      },
    },
    namespaceChangeButtonInput: {
      textAlign: "center",
    },
    namespaceChangeButtonInputFocused: {
      textAlign: "center",
    },
    namespaceChangeButtonPopupIndicator: {
      backgroundColor: "transparent",
      "&:hover": {
        backgroundColor: "transparent",
      }
    },
```

Reply:
Super helpful, thank you! ⭐️